packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.Lazy.Example.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { CommandBar, ICommandBarItemProps } from 'office-ui-fabric-react/lib/CommandBar';
+import { useConstCallback } from '@uifabric/react-hooks';
+import { ContextualMenuItemType } from '../../ContextualMenu';
+
+export const CommandBarLazyExample: React.FunctionComponent = () => {
+  const [menuItems, setMenuItems] = React.useState<ICommandBarItemProps[] | undefined>(undefined);
+
+  const onMenuDismissed = useConstCallback(() => {
+    setMenuItems(undefined);
+  });
+
+  const timeoutRef = React.useRef<NodeJS.Timeout>();
```

Comment:
This should be a number not a NodeJS.Timeout. Unfortunately the return type for setTimeout gets messed up since we have to include the node typings in the package for test purposes. So you'll have to cast when calling setTimeout later (unless you can find a better solution).